### ISSUE TITLE
Heatmap breaks metadata layout

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1787]
The hidden `<dt>` for the heatmap seems to be breaking the layout for the remainder of the metadata.

I think there's an argument for displaying a label for the heatmap, since there's nothing to announce it to a SR. (Okay, the a11y label is tenuous, but I'm adding it anyway.)

https://exhibits.stanford.edu/radarfilm/catalog/jt744vh3080

**Current**
<img width="1031" alt="Screen Shot 2020-02-27 at 10 14 27 AM" src="https://user-images.githubusercontent.com/6945691/75473716-8dd1a480-594a-11ea-96bc-66d6e2febc54.png">

**Label displayed**
<img width="1109" alt="Screen Shot 2020-02-27 at 10 14 42 AM" src="https://user-images.githubusercontent.com/6945691/75473729-94f8b280-594a-11ea-9290-a26b79b49369.png">

**Comment by ggeisler** [comment]
It would also be great if we could also remove that ~50px left padding on the map so it left aligns with the beginning of the text values.

**Comment by mejackreed** [comment]
I would suggest maybe that the map should be separate from the displayed metadata values entirely. This might help with a11y issue since it doesn't really semantically fit logically in that part and would give us the flexibility of a better display.

**Comment by ggeisler** [comment]
If we can do that and fix the a11y issue, that sounds good to me. I believe that's what we were trying to do (make it appear separate from the metadata list) by suppressing the metadata label for the map.

**Comment by jvine** [comment]
I can see separating it from the metadata but I do think there’s value in giving the map a visible label.

**Comment by mejackreed** [comment]
Dropping some alternatives to how things get displayed. Maybe this is a larger design question around the heatmaps plugin in general?

https://geo.btaa.org/catalog/06e7f566-852a-4914-929d-1bef38132eba
![Screen Shot 2020-03-03 at 2 12 47 PM](https://user-images.githubusercontent.com/1656824/75820193-24da9a00-5d59-11ea-9df8-e6e9baada3ce.png)
